.github/workflows/pr_ci.yml

```diff
@@ -31,6 +31,12 @@ jobs:
       # warnings and other better approaches don't work, see https://github.com/OrchardCMS/OrchardCore/pull/16317.
       run: |
         dotnet build -c Release -warnaserror /p:TreatWarningsAsErrors=true /p:RunAnalyzers=true /p:NuGetAudit=false
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Set Git line ending configuration
+      run: git config --global core.autocrlf true
```

Comment:
This shouldn't be necessary under Linux: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

Reply:
without this the command `dotnet format` will create differences and fail.